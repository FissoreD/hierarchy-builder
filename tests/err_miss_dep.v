From HB Require Import structures.
HB.mixin Record IsAddComoid A := {}.
HB.structure Definition AddComoid := { A of IsAddComoid A }.
HB.mixin Record IsAbelianGrp A of IsAddComoid A := {}.
HB.structure Definition AbelianGrp := { A of IsAbelianGrp A }.
Fail HB.mixin Record IsRing K of IsAbelianGrp K (*& IsAddComoid K*) := {}.
