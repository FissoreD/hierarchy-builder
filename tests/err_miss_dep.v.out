The command has indeed failed with message:
HB: Unable to find mixin err_miss_dep_IsAddComoid on subject K
